serverlessworkflow/modules/ROOT/pages/cloud/operator/developing-workflows.adoc

```diff
@@ -138,7 +138,7 @@ For example using VSCode, there are the commands needed:
 
 [source,bash,subs="attributes+"]
 ----
-curl -S https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sw.kogito_v1alpha08_sonataflow_devmode.yaml > workflow_devmode.yaml
+curl -S https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sonataflow.org_v1alpha08_sonataflow_devmodeWithConfigMapAndExternalResource.yaml > workflow_devmode.yaml
```

Comment:
@ricardozanini I'm wondering whether we shouldn't add a pure devmode sample too. This should work too, but for some reason it references a config map which is not used by the workflow definition, see [here](https://github.com/kiegroup/kogito-serverless-operator/blob/bedb0e816347a6e56a0a94d70489cc19f29c747f/config/samples/sonataflow.org_v1alpha08_sonataflow_devmodeWithConfigMapAndExternalResource.yaml#L37-L70).

Reply:
This YAML is just for reference, it will be deleted. I'll fix the PR.